libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab.component.ts

```diff
@@ -136,6 +138,10 @@ export class DhUsersTabComponent {
     this.userRolesStore.getRoles();
 
     this.onSearchInput();
+
+    this.profileModalService.onProfileUpdate$
+      .pipe(takeUntilDestroyed(this.destroyRef))
```

Comment:
Suggestion: No need for this.destoryRef in constructor

Reply:
Nice catch. Should be fixed now.